Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-auth.tsx

```diff
@@ -85,6 +87,7 @@ const AuthProvider = ({ user, children }: IAuthProviderProps) => {
 
     const value = {
         user: internalUser,
+        loginErrorMessage,
```

Comment:
As discussed before, this should be refactored to `state` & `actions`  when we do changes to this value.